src/vm/codeman.cpp

```diff
@@ -4166,7 +4166,13 @@ PCODE ExecutionManager::GetCodeStartAddress(PCODE currentPC)
     EECodeInfo codeInfo(currentPC);
     if (!codeInfo.IsValid())
         return NULL;
-    return (PCODE)codeInfo.GetStartAddress();
+    TADDR startAddr = codeInfo.GetStartAddress();
+
+#ifdef _TARGET_ARM_
+    if (startAddr != NULL)
```

Comment:
This is never going to be NULL. It is guaranteed by the `codeInfo.IsValid()` check above.